rotary_jet_spinning/datas/data.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Source : A simple model for nanofiber formation by rotary jet-spinning
+by mellado et al.
+
+Goal : Prediction of the radius of the fiber
+
+Change values of the machine and the polymer for which we want to run the code in deck.yaml.
+
+Inputs : Concerning the polymer : the surface tension, the density, the viscosity
+         Concerning the machine : the reservoir radius, the collector radius,
+                                  the orifice radius,
+                                  the angular viscosity of the spinneret.
+
+All data are in SI units.
+
+"""
+
+
+import numpy
+
+
+class Data:
+    
+    def __init__(self, deck, polymer, machine, model):
+        
+        self.deck = deck
+        self.polymer = polymer
+        self.machine = machine
+        self.model = model
+        self.final_radius_orifice_radius(deck, polymer, machine, model)
+        self.final_radius_omega(deck, polymer, machine, model)
+        self.final_radius_collector_radius(deck, polymer, machine, model)
+        self.final_radius_reservoir_radius(deck, polymer, machine, model)
+        self.final_radius_density(deck, polymer, machine, model)
+        self.final_radius_surface_tension(deck, polymer, machine, model)
+        self.final_radius_viscosity(deck, polymer, machine, model)
+        self.radius_x(deck, polymer, machine, model)
+        
+
+
+    def final_radius_orifice_radius(self, deck, polymer, machine, model):
+        
+        discretisation = int(deck.doc['Discretisation'])
+        # The higher the discretisation number is, the finer the discretisation will be,
+        # there will be more points on the graphic.
+        
+        orifice_radius = numpy.linspace(0.0001, 0.001, discretisation)
+        orifice_radius = orifice_radius.tolist()
+        
+        omega_th = []
+        initial_velocity = []
+        for l in range(discretisation):
+            omega_th.append(model.critical_rotational_velocity_threshold(polymer.surface_tension, orifice_radius[l],
+                                                           machine.reservoir_radius, polymer.density))
+            initial_velocity.append(model.Initial_velocity(omega_th[l], machine.reservoir_radius))
+        
+        nu = model.kinematic_viscosity(polymer.viscosity, polymer.density)
+        
+        Final_radius = []
+        for k in range(discretisation):
+            Final_radius.append(model.final_radius(orifice_radius[k], initial_velocity[k],
+                                            nu, machine.collector_radius, machine.omega))
+
+        return [orifice_radius, Final_radius]
+        
+
+    
+    def final_radius_omega(self, deck, polymer, machine, model):
```

Comment:
omega is the angular velocity. In the deck.yaml it is named as such. 
I think that for consistencies, the name should be keep the same. If possible replace omega by angular_velocity where applicable. It won't be pretty in some places, but it will maintain consistency and help readability i think.